### ISSUE TITLE
ARIA-describedby updates needed on form elements

### ISSUE THREAD

**Issue by cehfisher** [thememachine/eleven#196]
# The issue is:
You have pieces of code in your form elements that include code like `aria-describedby="edit-current-pass--description"` but there is no corresponding description. This renders the `aria-describedby` to be pointless.

# Screenshots:
![aria-missing](https://user-images.githubusercontent.com/15240149/29222608-d0bc8570-7e88-11e7-9eb9-0beaff095bea.png)

# Where (url)
http://drupalvm-eleven.dev/user/1/edit

# How to Fix:
If there is a field where the description is set for forms, make sure it is a required field. If the description is created more dynamically, make sure the markup includes the corresponding description. See this example:
```
<input id="last-name" type="text" name="textfield" aria-describedby="error-message" required>
<div id="error-message" class="visuallyhidden">This field is required.</div>
```
found in the A11Y style guide - http://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-text-fields

**Comment by mortendk** [comment]
oooh snap an old irritation of all of the ids made this happen- now they are back in description with more a11y love
